packages/data/src/useWalkthroughData.ts

```diff
@@ -200,7 +200,7 @@ export enum ResultCalculationType {
 
 export enum ResultLogicTypes {
   Fallback = "fallback",
-  GreaterThan = "greaterThan",
+  GreaterThanOrEqual = "greaterThanOrEqual",
```

Comment:
Double checking: We don't need a normal greater than for any of the results logic types?

Reply:
Correct! This is the only section that has "resultLogicType" in it. It's a one off for the table value retrieval in the result.